cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -13,23 +13,386 @@ License: CC-BY 4.0
+reports, or not documenting fixed vulnerabilities.
+
+The law requires the Steward to take actions, but clearly excludes any fees in the case of non-respect of the
+regulation.
+
+References: Article 24(2), Article 64(10b), Article 52(3)
+
+        Cyber Resilience Act, Article 24(2):
+        
+        Open-source software stewards shall cooperate with the market surveillance authorities, at their request,
+        with a view to mitigating the cybersecurity risks posed by a product with digital elements qualifying as
+        free and open-source software.
+
+        Further to a reasoned request from a market surveillance authority, open-source software stewards shall
+        provide that authority, in a language which can be easily understood by that authority, with the
+        documentation referred to in paragraph 1, in paper or electronic form.
+        
+        Cyber Resilience Act, Article 64(10) "Penalties":
+        
+        By way of derogation from paragraphs 3 to 9, the administrative fines referred to in those paragraphs shall not apply to the following:
+        [...] (b) any infringement of this Regulation by open-source software stewards.
+        
+        Cyber Resilience Act, Article 52 (3) "Market surveillance and control of products with digital elements in the Union market":
+
+        The market surveillance authorities designated under paragraph 2 of this Article shall also be responsible for carrying out
+        market surveillance activities in relation to the obligations for open-source software stewards laid down in Article 24.
+        Where a market surveillance authority finds that an open-source software steward does not comply with the obligations set
+        out in that Article, it shall require the open-source software steward to ensure that all appropriate corrective actions
+        are taken. Open-source software stewards shall ensure that all appropriate corrective action is taken in respect of their
+        obligations under this Regulation.
+        
+What Stewards and Project will need:
+- Know what organization is their market surveillance authority
+
+What resources could be created to help fulfilling that requirement:
+- A guide on finding out which is the appropriate market surveillance authority for each Steward
+- Verifying with market surveillance if they accept policies and communication in English
+
+### Mandatory reporting of exploited vulnerabilities and security incidents
+
+Stewards have obligation for mandatory reporting of exploited vulnerabilities and severe incidents. However, this obligation
+is more limited in scope that the one of manufacturers.
+
+First, Stewards are expected to report exploited vulnerabilities if they are involved in the development of the product.
+It means that is the Steward is handling other elements that development (for example the Project is taking decisions independently),
+they do report. 
```

Comment:
There appear to a a number of little errors here that is making it very hard for me to parse this sentence. Given the context, I think the intended meaning is:

"This means that if the Steward is handling other elements than development (for example the Project is taking decisions independently), the Steward does not report but the Project does.

Reply:
fixed thanks!